superset-frontend/src/features/alerts/components/AlertReportCronScheduler.tsx

```diff
@@ -66,9 +70,45 @@ export const AlertReportCronScheduler: React.FC<AlertReportCronSchedulerProps> =
 
     return (
       <>
-        <Radio.Group onChange={handleRadioButtonChange} value={scheduleFormat}>
-          <div className="inline-container add-margin">
-            <Radio data-test="picker" value="picker" />
+        <StyledInputContainer>
+          <div className="control-label">
+            {t('Schedule type')}
+            <span className="required">*</span>
+          </div>
+          <div className="input-container">
+            <Select
+              ariaLabel={t('Schedule type')}
+              placeholder={t('Schedule type')}
+              onChange={(e: any) => {
+                setScheduleFormat(e);
+              }}
+              options={SCHEDULE_TYPE_OPTIONS}
+            />
+          </div>
+        </StyledInputContainer>
+
+        <StyledInputContainer
+          data-test="input-content"
+          className="styled-input"
+        >
+          <div className="control-label">
+            {t('Schedule')}
+            <span className="required">*</span>
+          </div>
+          {scheduleFormat === 'input' ? (
+            <Input
+              type="text"
+              name="crontab"
+              ref={inputRef}
+              style={error ? { borderColor: theme.colors.error.base } : {}}
+              placeholder={t('CRON expression')}
+              disabled={scheduleFormat !== 'input'}
```

Comment:
Can it ever be disabled? We're rendering Input only when `scheduleFormat === 'input'`